### DISCUSSION TITLE
Frame Replication with Time-Aware Shaping - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#783] [Showcases]
Discuss the "Frame Replication with Time-Aware Shaping" showcase here. Ideas and recommendations are welcome.

**Comment by Lukas327** [comment]
Can you explain what does this sentence in the .ned file intend to do?

module LocalTsnSwitch extends TsnSwitch
{
    @defaultStatistic("gateStateChanged:vector"; module="eth[0].macLayer.queue.gate[0]");
}

**Comment by levy** [comment]
This is an undocumented feature of the Sequence Chart tool in the IDE. It tells the Sequence Chart tool to display the specified statistic on the axis of LocalTsnSwitch modules. So when you open the elog file, the chart will automatically display the gate state vector of the first gate of the first Ethernet interface for each LocalTsnSwitch in the network.
